packages/availity-workflow/scripts/proxy.js

```diff
@@ -54,8 +54,25 @@ function proxy() {
     config = [];
     // Iterate through each proxy configuration
     proxies.forEach(proxyConfiguration => {
+
+      const onProxyRes = function(proxyRes) {
+
+        if (proxyConfiguration.contextRewrite) {
+
+          Object.keys(proxyConfiguration.contextRewrite).forEach( regex => {
+            const value = proxyConfiguration.contextRewrite[regex];
+            const regexer = new RegExp(regex, 'g');
+            // Rewrite Locaion header else polling doesn't work
+            proxyRes.headers.location = proxyRes.headers.location.replace(regexer, value);
+            // TODO: rewrite JSON payload
```

Comment:
Not too sure if the payload is already parsed to JSON yet or not, but if it is, simply stringify it, `.replace(regexer, value)` it, and parse it back.